packages/itwin/grouping-mapping-widget/src/widget/components/GroupPropertyAction.tsx

```diff
@@ -28,11 +28,12 @@ import {
   SvgRemove,
 } from "@itwin/itwinui-icons-react";
 import type {
```

Comment:
whats up with the import being on multiple lines for no reason?

Reply:
comment applies for other files where imports were changed

Reply:
Oh, I see what happened. 
I accidently had prettier split it into multiple lines. 
```
import {
  ReportsApiConfig,
  ReportsApiConfigContext,
} from "../context/ReportsApiConfigContext";
```
ESLint then pulled the type out from the import verbatim I am assuming. 

```
import type {
  ReportsApiConfig} from "../context/ReportsApiConfigContext";
import {
  ReportsApiConfigContext,
} from "../context/ReportsApiConfigContext";
```